tests/datasets/test_mapinwild.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import builtins
+import os
+from pathlib import Path
+from typing import Any
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+from torch.utils.data import ConcatDataset
+
+from torchgeo.datasets import MapInWild
+
+pytest.importorskip("pandas", minversion="1.1.3")
+
+
+class TestMapInWild:
+    @pytest.fixture(params=["train", "validation", "test"])
+    def dataset(self, monkeypatch: MonkeyPatch) -> MapInWild:
+        md5s = {
+            "ESA_WC.zip": "3acbc2a2bc299e5a967f9349db484872",
+            "VIIRS.zip": "25081c93534c42c28d70ae67469c33ac",
+            "mask.zip": "e5042709de9ee97f83d14b3bb0a7bb78",
+            "s1_part1.zip": "c4732b3b9239983634dec2066fda11cf",
+            "s1_part2.zip": "50b72b5470ec6801e969cf292fec7d1d",
+            "s2_autumn_part1.zip": "2e4b7b09202504d1dc95c83a65685d8a",
+            "s2_autumn_part2.zip": "b8882dd9290124e1de4f0e0872774e6a",
+            "s2_spring_part1.zip": "c29184039ad7e7aee4a4bcae4e013bca",
+            "s2_spring_part2.zip": "9cb5a81804344c080591592828ba0a22",
+            "s2_summer_part1.zip": "8ce599ff71a9bfa2445cf460284fbec8",
+            "s2_summer_part2.zip": "7f5643b88c4b7395bb97f428d9573190",
+            "s2_temporal_subset_part1.zip": "89fcd50a65a7cfbbac61d91af3b44cb7",
+            "s2_temporal_subset_part2.zip": "5aa028759cefcb8b4fbe3da95e0f1ff1",
+            "s2_winter_part1.zip": "83a72480a8be070f8b256458e9a1a4f8",
+            "s2_winter_part2.zip": "fddd797d8bfd932e31a36a8423ff3704",
+        }
+
+        monkeypatch.setattr(MapInWild, "md5s", md5s)
+        root = os.path.join("tests", "data", "mapinwild")
+        transforms = nn.Identity()
+        modality = [
+            "mask",
+            "viirs",
+            "esa_wc",
+            "s2_winter",
+            "s1",
+            "s2_summer",
+            "s2_spring",
+            "s2_autumn",
+            "s2_temporal_subset",
+        ]  # noqa: E501
```

Comment:
Most of the noqa comments in this file can be removed. We generally want to break long lines anyway. The only exceptions are for long URLs where you should use this comment